src/tests/functional/inference_engine/transformations/common_optimizations/gru_cell_fusion.cpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <queue>
+
+#include "openvino/opsets/opset9.hpp"
+#include "transformations/common_optimizations/gru_cell_fusion.hpp"
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace std;
+using namespace ov;
+using namespace ov::opset9;
+using namespace ov::element;
+using namespace testing;
+
+namespace {
+
+enum class WeightsFormat {
+    zr,
+    rz
+};
+
+Output<Node> create_activation_by_name(const string& activation_name, const Output<Node>& input);
+
+    Output <Node> create_activation_by_name(const string &activation_name, const Output <Node> &input) {
+        if (activation_name == "sigmoid") {
+            return make_shared<Sigmoid>(input);
+        } else if (activation_name == "tanh") {
+            return make_shared<Tanh>(input);
+        } else if (activation_name == "relu") {
+            return make_shared<Relu>(input);
+        } else {
+            OPENVINO_ASSERT("Unsupported activation function");
+            return {};
```

Comment:
OpenVINO Assert should throw an exception. You don't need return here
```suggestion
            OPENVINO_ASSERT("Unsupported activation function");
```

Reply:
fixed

Reply:
error: control reaches end of non-void function [-Werror=return-type]